Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "RxMKMapView",
+    platforms: [.iOS(.v10)],
```

Comment:
Is there any reason this is `v10` rather than `v8` as specified in the Podfile?

Reply:
hmm i think any reason. will set v8 + bump rx swift